src/scripts/origins.js

```diff
@@ -62,11 +62,11 @@ export default {
     url: '*://*.bitrix24.com/*',
     name: 'Bitrix24'
   },
-  'breeze.pm': {
+  'app.breeze.pm': {
```

Comment:
I have mentioned all of these in the issue, but did you actually go thru all of them and checked whether their respective apps are actually under such subdomains?

Cause otherwise, I'm really confused as to why this solved the problem.

But partially I can't remember what was wrong for those domains in the first place. 
Which is a mistake on my part that I haven't written it in the issue. 😞

Reply:
>  but did you actually go thru all of them and checked whether their respective apps are actually under such subdomains?

The problem wasn't this. The button was getting the permission to run on these domains, the only issue is that the integrations was wrongly showing the checkboxes as unchecked. Note that the permission is set based on the `url` not on this key.

Reply:
But if that is the case, what's then the purpose of this change?

Reply:
To make button settings recognize this integration based on the key. If you see the file will you see that the key matches the domain of the integration but that doesn't necessarily mean that they weren't working is just that logic behind the checkboxes rely on this `key -> url` match to handle the state.